.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "suegithub"]
+	path = travis/ZNonPublicDeps
+	url = https://github.com/smootoo/ZNonPublicDeps.git
```

Comment:
Why are the drivers stored encrypted? Are we not allowed to distribute them? Where is the password for decryption stored? Also, if we want to go this route, we probably want to move this over to the slick org first, not have slick depend on an external github space.